### PR TITLE
add service domain to the sample work-cluster manifest

### PR DESCRIPTION
**Pull request by datianshi**
**What this PR does / why we need it**:
Did a pull request https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/559 got merged... 
But did update the workload cluster sample manifest. This will fix it.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ spec:
     services:
       cidrBlocks:
       - 100.64.0.0/13
+    serviceDomain: cluster.local
```
Comment (andrewsykim) [inline docs/getting_started.md]: I believe this is the default service domain, was that not the case in your cluster?

Comment (andrewsykim) [reply docs/getting_started.md]: ah nvm, it should be part of the generated manifests now

Comment (andrewsykim) [reply docs/getting_started.md]: CI is saying there's a trailing space here, can you fix it please?

**Comment by datianshi** [comment]
Fixed the trailing space

**Pull request merged**